File: DVLD_Project/Form11.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using BussnessLayerForDVLD;

namespace DVLD_Project
{
    public partial class Form11 : Form
    {
        int _N = 0;
        string _T = "";
        double _F = 0.0f;
        public Form11()
        {
            InitializeComponent();
        }

        public Form11(int N, string T, double F)
        {
            InitializeComponent();
            lblTypeID.Text = N.ToString();
            tbTypeName.Text = T.ToString();
            tbFees.Text = F.ToString();
            _N = N;
            _T = T;
            _F = F;
        }

        private void Form11_Load(object sender, EventArgs e)
        {

        }

        private void label4_Click(object sender, EventArgs e)
        {

        }

        private void pictureBox2_Click(object sender, EventArgs e)
        {

        }

        private void tbFirstName_TextChanged(object sender, EventArgs e)
        {

        }

        private void guna2Button1_Click(object sender, EventArgs e)
        {
            clsBussnesLayer.UpdateType(_N,tbTypeName.Text,Convert.ToDouble(tbFees.Text));
        }
    
    }
}

File: DVLD_Project/Form11.vb
Imports System
Imports System.Windows.Forms
Imports BussnessLayerForDVLD

Namespace DVLD_Project
    Public Partial Class Form11
        Inherits Form
        Private _N As Integer = 0
        Private _T As String = ""
        Private _F As Double = 0.0F
        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub New(N As Integer, T As String, F As Double)
            InitializeComponent()
            lblTypeID.Text = N.ToString()
            tbTypeName.Text = T.ToString()
            tbFees.Text = F.ToString()
            _N = N
            _T = T
            _F = F
        End Sub

        Private Sub Form11_Load(sender As Object, e As EventArgs)

        End Sub

        Private Sub label4_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub pictureBox2_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub tbFirstName_TextChanged(sender As Object, e As EventArgs)

        End Sub

        Private Sub guna2Button1_Click(sender As Object, e As EventArgs)
            clsBussnesLayer.UpdateType(_N, tbTypeName.Text, Convert.ToDouble(tbFees.Text))
        End Sub

    End Class
End Namespace
